src/app/components/Profile/Profile.tsx

```diff
@@ -0,0 +1,73 @@
+import { useEffect, useState } from 'react';
+import defaultAvatar from './defaultAvatar.svg';
+import classes from './Profile.module.css';
+
+type Profile = {
+  username: string | null;
+  displayName: string | null;
+  avatar: string | null;
+};
+
+function openDialog() {
+  overwolf.profile.openLoginDialog();
+}
+
+function Profile(): JSX.Element {
+  const defaultProfile: Profile = {
```

Comment:
I would declare this variable outside of the react component, but i can't explain why. :)

Reply:
see leons comment above 😉